src/views/v2/month/calendar-header.php

```diff
@@ -16,18 +16,18 @@
 $days_of_week = tribe_events_get_days_of_week();
 global $wp_locale;
 ?>
-<header role="rowgroup">
+<header class="tribe-events-calendar-month__header" role="rowgroup">
 
-	<h2 class="tribe-common-a11y-visual-hide" id="tribe-calendar-header"><?php printf( esc_html__( 'Calendar of %s', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
+	<h2 class="tribe-common-a11y-visual-hide" id="tribe-events-calendar-header"><?php printf( esc_html__( 'Calendar of %s', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 0d1111cfbb897e1c9b67a54b97cfa1c2</sub>